pkg/arvo/app/chat-cli.hoon

```diff
@@ -363,13 +365,20 @@
         ;~  (glue ace)
           (tag %create)
           security
-          ;~(plug path (punt ;~(pfix ace glyph)))
+          path
+          (punt ;~(pfix ace glyph))
+          (punt ;~(pfix ace ;~(pose (cold %| bar) (cold %& pad))))
         ==
         ;~((glue ace) (tag %delete) path)
         ;~((glue ace) (tag %invite) rw path ships)
         ;~((glue ace) (tag %banish) rw path ships)
       ::
-        ;~((glue ace) (tag %join) ;~(plug targ (punt ;~(pfix ace glyph))))
+        ;~  (glue ace)
+          (tag %join)
+          targ
+          (punt ;~(pfix ace glyph))
+          (punt ;~(pfix ace ;~(pose (cold %| bar) (cold %& pad))))
```

Comment:
Same as above.

I tried my hand at making an `(cmd-with-args %cmd ~[required] ~[optional])` parser building function but never could get it to compile. Should probably do another attempt, these are getting out of hand.

Reply:
I know very little about this type of parsing, so I was cargo-culting. Thanks for your suggestions.

Reply:
this is done